src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -66,14 +66,9 @@ def make_artifacts(
 
 @click.command()
 @click.argument("output_dir", type=click.Path(exists=True))
-@click.option(
-    "-V",
-    "--label-version",
-    type=str,
-    default=None,
-    show_default=True,
-    help="Provide a version number for final results. "
-    "Version should follow format of '#.#.#'.",
+@click.argument(
+    "label_version",
+    # Version for final results and metadata file. This should be of format '#.#.#'
```

Comment:
This isn't enforced. Should it?

Reply:
Yes it is https://github.com/ihmeuw/vivarium_census_prl_synth_pop/blob/main/src/vivarium_census_prl_synth_pop/tools/cli.py#L184